### ISSUE TITLE
Can’t connect to Foscam G2 using webpage mode. 

### ISSUE THREAD

**Issue by mrfatboy** [IPCams/BugTracker#49]
**Describe the bug**

Can’t connect to Foscam G2 using webpage mode. 

**To Reproduce**
Enter correct settings 

**Expected behavior**
Be able to connect

**Screenshots**
Here is a screenshot of the error. It looks like it’s trying to detect a plug-in required by a normal browser. 

The Fowlr app works correctly with same settings. 

![80BBE227-5C53-4D0D-A26D-7679854F7E63](https://user-images.githubusercontent.com/3344008/114683721-1b925f80-9cc5-11eb-84e8-43b0fca3e389.jpeg)

**App Version**
App version can be found at the bottom of the settings page. 2.3

**Device & OS Version**
List device (iPhone/iPad), model (XS, 11 Pro, etc.) and OS version (13.5, 14.0, etc.).

iPhone 11 iOS 14.2

**Additional context**

**Comment by tylerjonesio** [comment]
Do you happen to know if your camera's webpage works at all in safari? If Safari works okay, it should be able to work within IPCams. But if not, there's not much that can be done as it is limited to what Safari and iOS's underlying web technologies support. Which for some cameras works just fine, but I know not all will support Safari/iOS.

**Comment by mrfatboy** [comment]
The same result happens in Safari. However using the Fowlr Foscam app works using web mode.

**Comment by tylerjonesio** [comment]
That's interesting! I wonder what they're doing to override the default Safari (and in turn IPCams) behavior here. Since I don't have a Foscam camera to test against, I can't promise anything. But I'll look into it and see what I can do.

**Comment by mrfatboy** [comment]
Yes. Not sure either. On desktop browsers I download the plugin? To make it work. Some iOS apps just seem to work 🤷‍♂️
